_posts/2019-05-07-builds-without-bytes.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: posts
+title: "Faster remote builds in Bazel 0.25"
+authors:
+  - buchgr
+---
+We are excited to announce the initial availability of ["Remote Builds without the Bytes"](https://github.com/bazelbuild/bazel/issues/6862) in Bazel 0.25.
+
+ 
+When using Bazel with remote caching and execution it downloads every output file of every action executed (or cached) to the host machine. The total size of all output files in a build will often be in the tens and even hundreds of gigabytes and make the network bandwidth the limiting factor. We know from experience that most of the time developers are only interested in a small subset of the output files produced during a build i.e. the final binary and quite often in a continuous integration build the output files are of no interest at all.
+ 
+We have thus been working on virtualizing action outputs in Bazel so to only transfer file metadata, that is content hash and size, instead of file contents. This will significantly reduce the amount of data being transferred between Bazel and the remote caching or execution system. Our own testing and early user reports suggest that double digest build speed improvements are the common case. The specific gains for your project will mostly depend on the available network bandwidth, latency and size of build outputs.
```

Comment:
I can't parse this:  "that double digest build speed improvements "

Reply:
Ahh a typo. I rephrased it. Done.